src/sage/sets/set.py

```diff
@@ -1085,6 +1082,30 @@ def __richcmp__(self, other, op):
             return rich_to_bool(op, 0)
         return rich_to_bool(op, -1)
 
+    def isdisjoint(self, other):
```

Comment:
The standard in Sage is to add underscores between separate words in method names.

```suggestion
    def is_disjoint(self, other):
```

Reply:
I would suggest not to do this in this very special case, because `Set` is supposed to be a "better" `set` (it is not, in my opinion, but it tries), and it is `set.isdisjoint` rather than `set.is_disjoint`.

Reply:
Alright, I see.

Reply:
I'm afraid I have to take this back, the situation in sage is already chaotic :-(

`Set` (and `FrozenBitset`) indeed implements `issubset` etc., but there are some classes (`MutablePoset`, `ManifoldSubset`, `RealSet`, I couldn't find any others) that implement `is_subset`.  In #41113, another class is going to implement `is_subset`.

It is not clear to me what's best.

Reply:
I suppose fundamentally we should make a choice and then rename/deprecate. For the time being, implementing both could be alright (it's just an extra line `isdisjoint = is_disjoint` in the class).